dist/libnccl-net-ofi.spec

```diff
@@ -0,0 +1,83 @@
+# Whether to build with cuda support. Default: on if neuron
+%if "%{with_cuda}" == "1" && "%{with_neuron}" == "1"
```

Comment:
why duplicate checks that configure will do?

Reply:
I'm using this to generate the package name, but I'm not good enough at rpm macros to default to a sane string when neither is set.